tests/h/storage_test.py

```diff
@@ -210,6 +210,20 @@ def test_it_allows_when_unscoped_group(
 
         assert result == models.Annotation.return_value
 
+    def test_it_allows_outside_scope_when_enforce_scope_is_False(
+        self, pyramid_request, pyramid_config, group_service, scoped_open_group, models
+    ):
+        scoped_open_group.enforce_scope = False
+        group_service.find.return_value = scoped_open_group
+
+        data = self.annotation_data()
+        data["target_uri"] = "http://www.foo.com/boo/bah.html"
```

Comment:
So if I understand correctly this is setting `"target_uri"` to a URI that's outside of `scoped_open_group` scope? Which is the crux of the test, it would cause `create_annotation()` to fail if `enforce_scope` was `True`. I wonder if a code comment could be added to this line to clarify this as it seems to be the crucial line of the test.